mayastor/src/bdev/nexus/nexus_bdev_rebuild.rs

```diff
@@ -18,13 +18,13 @@ use crate::{
         nexus_child::ChildState,
     },
     core::Reactors,
-    rebuild::{RebuildJob, RebuildOperations, RebuildState},
+    rebuild::{ClientOperations, RebuildJob, RebuildState},
 };
 
 impl Nexus {
     /// Starts a rebuild job in the background
     pub async fn start_rebuild_rpc(&mut self, name: &str) -> Result<(), Error> {
-        self.start_rebuild(name).await?;
+        let _ = self.start_rebuild(name).await?;
```

Comment:
you sure you want to swallow whatever this returns?

Reply:
I think so because we don't want to return the rust channel on the rpc call